site/src/analytics-element.ts

```diff
@@ -10,7 +10,7 @@ import {
 import "./analytics-active-installations";
 import "./analytics-average";
 import "./analytics-integrations";
-import "./analytics-core-versions";
+import "./analytics-releases";
```

Comment:
Since this replaces `analytics-core-versions`, ./src/analytics-core-versions.ts should be removed.